scripts/circleci-jobs/openshift3-integration-tests.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/python3
+
+import os
+import subprocess
+
+DOCKERHUB_USER = os.getenv('DOCKERHUB_USER')
+DOCKERHUB_PASSWORD = os.getenv('DOCKERHUB_PASSWORD')
+PROCESS = subprocess.run(["docker", "login", "--username", DOCKERHUB_USER, "--password", DOCKERHUB_PASSWORD], capture_output=True, text=True)
+print(PROCESS.stdout)
+
+CIRCLE_BRANCH = os.getenv("CIRCLE_BRANCH")
+if CIRCLE_BRANCH == "staging":
+    IMAGE_TAG = "staging-candidate"
+else:
+    IMAGE_TAG = "discardable"
+CIRCLE_SHA1 = os.getenv("CIRCLE_SHA1")
+KUBERNETES_MONITOR_IMAGE_NAME_AND_TAG = "snyk/kubernetes-monitor:" + IMAGE_TAG + "-" + CIRCLE_SHA1
+
+PROCESS = subprocess.run(["docker", "pull", KUBERNETES_MONITOR_IMAGE_NAME_AND_TAG], capture_output=True, text=True)
+print(PROCESS.stdout)
+
+PROCESS = subprocess.run(["npm", "run", "test:integration:openshift3"], capture_output=True, text=True, env=dict(os.environ))
```

Comment:
it looks like this script does two things:
1. sets up the integration test environment by pulling the right image
2. wraps npm test

why not let it just handle the setup, and let the circleCI call npm test as it does for all other flows?